lite/examples/object_detection/ios/ObjectDetection/ModelDataHandler/ModelDataHandler.swift

```diff
@@ -52,21 +52,11 @@ class ModelDataHandler: NSObject {
 
```

Comment:
Rename `ModelDataHandler` to `ObjectDetectionHelper` to be more precise. Please also rename the folder and the filename accordingly.